skeleton/spec/spec_helper.rb

```diff
@@ -9,6 +9,19 @@
 fixture_path = File.expand_path(File.join(__FILE__, '..', 'fixtures'))
 module_name = File.basename(File.expand_path(File.join(__FILE__,'../..')))
 
+# Add fixture lib dirs to LOAD_PATH. Work-around for PUP-3336
+if Puppet.version < "4.0.0"
+  Dir["#{fixture_path}/modules/*/lib"].entries.each do |lib_dir|
```

Comment:
This logic should go away; [rspec-puppet added duplicate logic in mid-2015](https://github.com/rodjek/rspec-puppet/pull/237) and our modules are tested against Puppet 4+